jenkins/builder_info.rb

```diff
@@ -1,13 +1,13 @@
 #!/opt/chef/embedded/bin/ruby
 
 require 'rubygems'
-require 'yajl'
+require 'chef/json_compat'
 
 class OhNo
   def ohai
     if !@ohai
       raw = `ohai -lerror`
-      @ohai = Yajl::Parser.new.parse(raw)
+      @ohai = Chef::JSONCompat.from_json(raw)
```

Comment:
I think the safest method in Chef::JSONCompat is `parse` rather than `from_json` if we don't need the create_addition behavior (instantiation of ruby objects from json_class keys in a Hash rather than just using a hash). Do we need create_additions here?